django/db/models/functions/text.py

```diff
@@ -150,6 +150,22 @@ class LTrim(Transform):
     lookup_name = 'ltrim'
 
 
+class MD5(Transform):
+    function = 'MD5'
+    lookup_name = 'md5'
+
+    def as_oracle(self, compiler, connection, **extra_context):
+        return super().as_sql(
+            compiler,
+            connection,
+            template=(
+                "LOWER(RAWTOHEX(STANDARD_HASH(UTL_I18N.STRING_TO_RAW("
+                "%(expressions)s, 'AL32UTF8'), 'MD5')))"
```

Comment:
Wrap `%(expressions)s` in parentheses.

Reply:
Extra parentheses are nor required here.

Reply:
Not even if you pass an expression? Just trying to avert a common problem we've had to fix numerous times. If you think there is no expression that could be an issue, I'm happy to ignore this.

Reply:
IMO parentheses are not required, since we pass `expression` as an argument to the `UTL_I18N.STRING_TO_RAW()` function :thinking: . We don't wrap `expression` with extra parentheses in other Oracle's hooks e.g. [Reverse](https://github.com/django/django/blob/master/django/db/models/functions/text.py#L197-L200).